main.js

```diff
@@ -0,0 +1,52 @@
+// var userChoice = prompt('Would you like to play rock, paper or scissors?')
+var compScore = 1;
+var playerScore = 1;
+var computerMove = Math.random();
+
+function winCondition() {
+  if (compScore == 5) {
+  }
+}
+
+if (computerMove <0.33) {
```

Comment:
You can use an example from  https://developer.mozilla.org/ to convert the random number into integer (if you need it). 
function getRandomInt(max) {
  return Math.floor(Math.random() * Math.floor(max));
}